utils.go

```diff
@@ -402,34 +402,42 @@ func sanitizeRootfs(id, rootfs string) string {
 	return rootfs
 }
 
-// getLinuxHeaderMounts returns a list of read-only mounts of the host's linux kernel headers.
-func getLinuxHeaderMounts() ([]specs.Mount, error) {
+// getLinuxHeaderMounts returns a list of read-only mounts of the host's linux
+// kernel headers.
+func getLinuxHeaderMounts(
+	rootfs string,
+	sysDistro string,
```

Comment:
This param is not needed by this function right? I don't see it being used (which makes sense).

Reply:
Yep, already removed it, was a left over from a previous logic.